### PR TITLE
[GEN] New index 

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by ofranc — APPROVED** [review]

File: containers/kubernetes/index.mdx [hunk 1]
```diff
@@ -1,8 +1,93 @@
...
+    />
+    <SummaryCard
+        title="How-tos"
+        icon="help-circle-outline"
+        description="Check our guides to creating and managing Kubernetes clusters and their features."
+        label="View How-tos"
+        url="/containers/kubernetes/how-to/"
+    />
+    <SummaryCard
+        title="Additional resources"
```
Comment (bene2k1) [inline containers/kubernetes/index.mdx:40]:
```suggestion
        title="Additional content"
```

File: containers/kubernetes/index.mdx [hunk 2]
```diff
@@ -1,8 +1,93 @@
...
+        icon="book-open-outline"
+        description="Guides to help you configuring your cluster and its advanced configuration."
+        label="View additional resources"
```
Comment (bene2k1) [inline containers/kubernetes/index.mdx:43]:
```suggestion
        label="View additional content"
```

File: containers/kubernetes/index.mdx [hunk 3]
```diff
@@ -1,8 +1,93 @@
...
-content:
-  h1: Kubernetes Documentation
-  paragraph: Kubernetes Documentation
+  title: Kubernetes Documentation
+  description: Kubernetes Documentation
 ---
+
+<ProductHeader
+   productName="Kubernetes"
+   productLogo="k8sKapsule"
```
Comment (nerda-codes) [inline containers/kubernetes/index.mdx:9]: Il me semble qu'il y a "wire" à la fin du nom du logo, non ? => "k8sKapsuleWire"

**Review by nerda-codes — APPROVED** [review]

File: containers/kubernetes/index.mdx [hunk 4]
```diff
@@ -1,8 +1,93 @@
...
+        url="/tutorials/odoo-easy-deploy/"
+        label="Read more"
+    />
+</Grid>
+
+<ClickableBanner
+    productLogo="console"
+    title="Kubernetes API"
+    description="Learn how to create and manage your Kubernetes clusters through the API."
+    url="/developers/api/kubernetes/"
```
[the remark this answers is not in the document]
Comment (ofranc) [reply containers/kubernetes/index.mdx:77]: www.scaleway.com/en/developers/api/kubernetes/

**Review by ofranc — APPROVED** [review]

**Pull request merged**